modules/ipc/tests/action_server_tests.cpp

```diff
@@ -132,9 +149,22 @@ TEST(ActionServer, ActionServerStopRequest) {
       SERVICE_CALL_TIMEOUT);
 
   requested_started.wait(false);
-  auto success = requestActionServerToStopExecution(*client_session, action_server_data.topic_config);
-  EXPECT_TRUE(success);
 
+  // requested_started guarantee that the request is now being proceed, but, although the action server create
+  // the stop request service before processing the request, the stop service could still bootstrapping, as
+  // this is controlled by Zenoh.
+  // For this reason there is the chance that we need to try multiple times to stop the action server.
+  while (true) {
+    auto success = requestActionServerToStopExecution(*client_session, action_server_data.topic_config);
+    if (success) {
+      break;
+    }
+
+    std::this_thread::sleep_for(std::chrono::milliseconds(4));
+  };
+
+  const auto wait_res = reply_future.wait_for(REPLY_SERVICE_TIMEOUT);
```

Comment:
so this is the actual fix right? is the REPLY_SERVICE_TIMEOUT only for debugging reasons then?

Reply:
the actual fix is the `while(true)`, but the `wait_for` is anyway a good addition